lib/polyfill/patchedmediakeys_safari.js

```diff
@@ -563,8 +555,7 @@ shaka.polyfill.PatchedMediaKeysSafari.MediaKeySession.prototype.
   // We add the 'contentId' parameter below and let DrmEngine handle it.
   var event2 = new shaka.util.FakeEvent('message', {
     messageType: isNew ? 'licenserequest' : 'licenserenewal',
-    message: event.message.buffer,
-    'contentId': this.contentId_
```

Comment:
The application should still have a way of getting the `contentId`, but I think it could be an event from the player.